docs/source/troubleshooting.rst

```diff
@@ -0,0 +1,58 @@
+Troubleshooting
+##############################################################################
+
+``csfg`` helper script
+==============================================================================
+
+I get a ``No such file or directory`` error when running the ``csfg`` script
+------------------------------------------------------------------------------
+
+Check your terminal working directory is within the ``cs-field-guide`` directory,
+the root directory of the project.
+Running the ``ls`` command in this directory should list the ``csfg`` file.
+
+I have an error when running ``./csfg start``
+------------------------------------------------------------------------------
+
+If you are having issues running the ``start`` command, try rebuilding the
+system images with ``./csfg build``.
+Changes may be have been made to the system images since you initally created
+them.
+
+If this still doesn't solve your problem, you could also try deleting any
+existing images with ``./csfg wipe``, and then build and start the system with
+``./csfg start``.
```

Comment:
`./csfg wipe` is broken currently, we should probably make an issue for it

Reply:
Is it broken or is it just giving an error because there are some running containers it won't remove?

Reply:
run end first, then it'll break with a terminal syntax error

Reply:
Turns out I had CSU conatiners running.. will make an issue

Reply:
Should I say that it's currently broken in the docs and link to the issue?

Reply:
I might just leave as is in the docs for now. Will update if we think the issue is going to take time to resolve.